lib/pdffield.js

```diff
@@ -165,7 +165,11 @@ let PDFField = (function PDFFieldClosure() {
             }
         }, _getFieldBaseData.call(this, box));
 
-        this.Boxsets.push({boxes:[anData]});
+        if(box.fieldValue) {
+            anData.checked = !(box.fieldValue.match(/off/i));
```

Comment:
if you intend to only match the case-insensitive word "off", then you should use `/^off$/`. Otherwise you'll match any word that has "off" in it. Maybe that doesn't matter because the options are only "off" and "on".